### PR TITLE
feat: Provide basic setup

### PR DESCRIPTION
**Pull request by tomkerkhove**
Provide basic setup:
- Provide decent organization README
- Provide decent README
- Remove unused resources
- Keep issue templates to make them available in every repo automatically

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/blogpost-proposal.md [hunk 1]
```diff
@@ -1,20 +0,0 @@
----
-name: Blogpost Proposal
```
Comment (stijnmoreels) [inline .github/ISSUE_TEMPLATE/blogpost-proposal.md:2]: Don't we use this? I think I used this one before.

**Review by stijnmoreels — APPROVED** [review]
Not sure why we had a source repo in here. Maybe something was requiring source code in the past?
Thx for this.

File: .github/ISSUE_TEMPLATE/blogpost-proposal.md (hunk 1 above)
Comment (tomkerkhove) [reply .github/ISSUE_TEMPLATE/blogpost-proposal.md:2]: Do we have this in every component repo? If so, I can re-add it.

**Comment by tomkerkhove** [comment]
> Not sure why we had a source repo in here. Maybe something was requiring source code in the past?

This is because of the GitHub template that I have used to create the repo.

File: .github/ISSUE_TEMPLATE/blogpost-proposal.md (hunk 1 above)
Comment (stijnmoreels) [reply .github/ISSUE_TEMPLATE/blogpost-proposal.md:2]: Ah, like this, Ah, no, I mistaken the kind of repo I'm reviewing 😄 My bad.

**Comment by stijnmoreels** [comment]
> > Not sure why we had a source repo in here. Maybe something was requiring source code in the past?
> 
> This is because of the GitHub template that I have used to create the repo.

Aha, was confused with the repo I was reviewing, 😄  my bad.

**Pull request merged**